components/eamxx/src/physics/nudging/eamxx_nudging_process_interface.cpp

```diff
@@ -525,19 +508,20 @@ void Nudging::run_impl (const double dt)
         //
         auto nudging_weights_field = get_helper_field("nudging_weights");
         // appply the nudging tendencies to the ATM states
-        apply_weighted_tendency(atm_state_field, int_state_field, nudging_weights_field, dt);
+        apply_weighted_tendency(atm_state_field, field_after_vinterp, nudging_weights_field, dt);
       } else if (m_refine_remap_vert_cutoff > 0.0) {
         // If we have a cutoff, we apply the tendency with p_mid cutoff
         // First, get p_mid the field in the atm (i.e., int) state
         auto p_mid_field = get_field_in("p_mid");
         // Then, call the tendency with a Heaviside-like cutoff
-        apply_vert_cutoff_tendency(atm_state_field, int_state_field,
-                                   p_mid_field, m_refine_remap_vert_cutoff, dt);
+        apply_vert_cutoff_tendency(atm_state_field, field_after_vinterp,
+                                   p_mid_field, dt);
       } else {
-        apply_tendency(atm_state_field, int_state_field, dt);
+        apply_tendency(atm_state_field, field_after_vinterp, dt);
       }
     }
   }
+  m_atm_logger->info("Running nudging...done!");
```

Comment:
You mean adding a time stamp to the logger message?

Reply:
I think we should add some logging capability to the time interpolation struct, so that it can log which time slices it's interpolating (if 2+ data files, maybe also print which files it's currently reading).